presets/4.3/other/ELRS_Passthrough.txt

```diff
@@ -0,0 +1,14 @@
+#$ TITLE: ExpressLRS Passthrough Betaflight Settings
+#$ FIRMWARE_VERSION: 3.5
```

Comment:
are you sure it works with 3.5? why not 4.0 or 4.1?

Reply:
I don't have anything on 4.0 or 4.1 to test.  Not a great reason, but I can't see why it wouldn't.

Reply:
what about 3.5? Does it need to be there/does it works there?

Reply:
I have tested with 3.5 - it also works with that.  Does not need to be there, but I have performed that testing.